bots/bot.go

```diff
@@ -0,0 +1,36 @@
+package bot
+
+import "crypto/tls"
+
+type BotHandleFunc func(string, []byte)
+
+type Bot interface {
+	HandleRegistration(string, []byte) string
+	Run()
+	Stop()
+}
+
+func SendRequestToCONIKS(address string, msg []byte) ([]byte, error) {
+	conf := &tls.Config{
+		InsecureSkipVerify: true,
```

Comment:
Yes, you're right. But that chance for they to have access to both TLS certs is so small, I think. Hopefully, this would be an extra security layer for our implementation.